specification/communication/data-plane/Identity/stable/2022-10-01/CommunicationIdentity.json

```diff
@@ -36,7 +36,7 @@
           "default": {
             "description": "Error",
             "schema": {
-              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+              "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationErrorResponse"
```

Comment:
Changes in the common swagger: 
methods returning the CommunicationIdentifierModel need to return a string property kind which can be one of the following four values: communicationUser, phoneNumber, microsoftTeamsUser, or unknown.

Identity SDK returns only Communication Identities (not PhoneNumbers or other types of identifiers).
The only model from Common that this swagger utilizes is the CommunicationErrorResponse.
Therefore, the common swagger changes have no impact.